src/networking.tf

```diff
@@ -59,3 +59,15 @@ module "local_network_gateways" {
     resource_groups = module.resource_groups
   }
 }
+
+module "private_dns_zones" {
+  source   = "./modules/_networking/private_dns_zone"
+  for_each = var.private_dns_zones
+
+  global_settings = var.global_settings
```

Comment:
this must be `local.global_settings` for consistency with the rest of the modules